utils/context_processor.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+
+
+def google_analytics(request):
+    """
+    Use the variables returned in this function to
+    render your Google Analytics tracking code template.
+    """
+    host = request.get_host()
```

Comment:
What happens when get_host() method fails? The documentation says that it could. (Ref https://docs.djangoproject.com/en/1.9/ref/request-response/#django.http.HttpRequest.get_host)